Day22_Manipulating_DOM_level1_solution/numberDisplay.js

```diff
@@ -0,0 +1,37 @@
+
+
+const btn = document.querySelector(".btn1");
+const box = document.querySelector(".container");
+let i=0, is_prime;
+
+function start(){
+    for(i=0; i<26; i++){
+
+        const newdiv = document.createElement("div");
+        newdiv.innerText = i;
+        newdiv.classList.add("class1");
+

```

Comment:
Well, nice... but here's a little fun challenge for you!
Can you make this code efficient? [hint: finding primes with root ]